src/checkers/athena.rs

```diff
@@ -0,0 +1,48 @@
+use crate::checkers::checker_result::CheckResult;
+use lemmeknow::Identify;
+
+use super::{
+    checker_type::{Check, Checker},
+    english::EnglishChecker,
+    human_checker,
+    lemmeknow_checker::LemmeKnow,
+};
+// use CheckerTypes;
+pub struct Athena;
+
+impl Check for Checker<Athena> {
+    fn new() -> Self {
+        Checker {
+            // TODO: Update fields with proper values
+            name: "Athena Checker",
+            description: "Runs all available checkers",
+            link: "",
+            tags: vec!["athena", "all"],
+            expected_runtime: 0.01,
+            popularity: 1.0,
+            lemmeknow_config: Identify::default(),
+            _phatom: std::marker::PhantomData,
+        }
+    }
+
+    fn check(&self, text: &str) -> CheckResult {
+        // TODO: wrap all checkers in oncecell so we only create them once!
```

Comment:
Can you explain like I'm 5 what OnceCell does? https://github.com/matklad/once_cell <3

Reply:
The main goal behind using it is that we can initialize out decoder only once and get reference to them whenever we want. 
[like this](https://docs.rs/once_cell/1.13.0/once_cell/sync/struct.OnceCell.html#method.get_or_init). 

We can put the declaration inside a `static`, so that we don't initialize Decoder every time  we call `check`.